From 411a03f149802f26229c0fa8883bb5964fabcdcc Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b32804@freescale.com>
Date: Mon, 28 May 2012 15:00:46 +0800
Subject: [PATCH 1562/2463] ENGR00210677: [MX60_ARM2]SDIO-WIFI: can not work.

enhance drive strength of sd pad ctrl to 40ohm otherwise it
will report error while trying to communicate with atheros
wifi.

error message looks like:
 - "mmc2: error -84 whilst initialising SDIO card"

Signed-off-by: Ryan QIAN <b32804@freescale.com>
---
 arch/arm/plat-mxc/include/mach/iomux-mx6sl.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h b/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h
index 529ee99..5ce2318 100755
--- a/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-mx6sl.h
@@ -37,7 +37,7 @@
 
 #define MX6SL_USDHC_PAD_CTRL (PAD_CTL_PKE | PAD_CTL_PUE |		\
 		PAD_CTL_PUS_22K_UP  | PAD_CTL_SPEED_LOW |		\
-		PAD_CTL_DSE_80ohm   | PAD_CTL_SRE_FAST  | PAD_CTL_HYS)
+		PAD_CTL_DSE_40ohm   | PAD_CTL_SRE_FAST  | PAD_CTL_HYS)
 
 #define MX6SL_USDHC_PAD_CTRL_100MHZ (PAD_CTL_PKE | PAD_CTL_PUE |	\
 		PAD_CTL_PUS_47K_UP  | PAD_CTL_SPEED_MED |		\
-- 
1.7.7.4

